### PR TITLE
Enhancement/add a polling system

### LINKED ISSUE #191: add a poling system to refresh page every set time

### PR DESCRIPTION
**Pull request by martiuh**
CHANGES

- convert `NavBar.jsx` to hooks
- rename `NavBarContainer.jsx` to `ǸavBarWrapper.jsx`
- add a polling system to `CalendarContainer.jsx`

closes #191

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -1,43 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavBarRightSide } from "./NavBarRightSide";
-import { NavBarContainer } from "./NavBarContainer";
+import { NavBarContainer } from "./NavBarWrapper";
```
Comment (mishelashala) [inline src/components/NavBar/NavBar.jsx]: Component must have the same name of the file in which is contained:

```js
import { NavBarContainer } from "./NavBarContainer";
```

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -1,43 +1,31 @@
...
-  };
+  const handleClose = () => updateAnchorEl(null);
 
-  render() {
-    const { anchorEl } = this.state;
-    const {
-      authContext: { onLogout }
-    } = this.props;
-    const username = storageService.getUserName();
+  const username = storageService.getUserName();
```
Comment (mishelashala) [inline src/components/NavBar/NavBar.jsx]: maybe we could have a storage hook:

```js
const [username, updateUsername] = useStorage('username');
```

what do you think?

File: src/pages/Room/CalendarContainer.jsx [hunk 3]
```diff
@@ -1,77 +1,114 @@
...
+import React, { useState, useContext, useEffect, useRef } from "react";
 import { withRouter } from "react-router-dom";
 import { bookingService, roomService } from "services";
 import { Calendar } from "./Calendar";
 import { Error500 } from "pages/Error500";
 import { getUTCDateFilter } from "utils/BookingFilters";
 import { AuthContext } from "../../context/AuthContext";
 
+// https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+const useInterval = (callback, delay) => {
```
Comment (mishelashala) [inline src/pages/Room/CalendarContainer.jsx]: move `useInterval` to its own file, maybe inside `src/hooks/useInterval`

File: src/components/NavBar/NavBar.jsx (hunk 2 above)
Comment (martiuh) [reply src/components/NavBar/NavBar.jsx]: Yes, it's a good idea. less requests to the server (since the polling system makes a request every 1.5 secs)

**Pull request merged**